fpdf/fpdf.py

```diff
@@ -56,7 +56,7 @@
 LOGGER = logging.getLogger(__name__)
 
 # Global variables
-FPDF_VERSION = "2.2.0"
+FPDF_VERSION = "2.2.0rc2"
```

Comment:
Maybe the `Last update` comment on line 6 could be removed, as I fear we will always forgot to update it...

Reply:
yeah im fine with that if you want to do that, you could push to this branch and then ill approve